clap_generate/Cargo.toml

```diff
@@ -0,0 +1,50 @@
+[package]
+name = "clap_generate"
+version = "3.0.0-beta.1"
+edition = "2018"
+authors = [
+	"Kevin K. <kbknapp@gmail.com>",
+	"Pavan Kumar Sunkara <pavan.sss1991@gmail.com>",
```

Comment:
We exist as well :p

Reply:
I have been meaning to just remove all the names and put "Clap Contributors". What do you think?

Reply:
That was going to be my next suggestion. The Clap Team or Clap Maintainers

Reply:
I would like to propose leave @kbknapp - as original author of these crates - and @TeXitoi (in `clap_derive/Cargo.toml`) - as original author of the derive; and then put "Clap Contributors". I want to highlight them because they have (or had) been putting admirable amount of efforts into these crates from the very beginning, even thought the crates weren't so hypy and popular back then.

Reply:
That sounds fine to me as well. I'd rather go with Maintainers though than contributors. There's a subtle difference